snippets/median.md

```diff
@@ -0,0 +1,23 @@
+---
+title: median
+tags: math,median,beginner
+---
+Find the median of a list of numbers
+
+Sort the numbers of the list using the sort function and find the median, which is the middlemost element of the list.
+
+```py
+def median(list):
+	list.sort() # The sort function of python
+	list_length = len(list)
+	if list_length%2==0:
```

Comment:
You can probably simplify this by moving the list length in the calculation or doing something like moving the if else inside the calculation somehow, minor issue, if you can't fix it, let it be.